docs/installation/integrations/react-multiroot.md

```diff
@@ -0,0 +1,139 @@
+* `onFocus` &ndash; A function called when the editor was focused. See the {@link module:engine/view/document~Document#event:focus `editor.editing.view.document#focus`} event.
+* `onError` &ndash; A function called when the editor has crashed during the initialization or during the runtime. It receives two arguments: the error instance and the error details.
+  Error details is an object that contains two properties:
+	* `{String} phase`: `'initialization'|'runtime'` &ndash; Informs when the error has occurred (during the editor or context initialization, or after the initialization).
+	* `{Boolean} willEditorRestart` &ndash; When `true`, it means that the editor component will restart itself.
+
+The editor event callbacks (`onChange`, `onBlur`, `onFocus`) receive two arguments:
+
+1. An {@link module:utils/eventinfo~EventInfo `EventInfo`} object.
+2. An {@link module:editor-multi-root/multirooteditor~MultiRootEditor `MultiRootEditor`} instance.
+
+## Hook values
+
+The `useMultiRootEditor` hook returns the following values:
+
+* `editor` &ndash; The instance of created editor.
+* `toolbarElement` &ndash; `ReactElement` that contains the toolbar. It could be rendered anywhere in the application.
+* `editableElements` &ndash; An array of `ReactElements` that describes the editor's roots. This array is updated after detaching an existing root or adding a new root.
+* `data` &ndash; The current state of the editor's data. It is updated after each editor update. Note that you should not use it if you disabled two-way binding by passing the `disableTwoWayDataBinding` property.
+* `setData` &ndash; The function used for updating the editor's data.
+* `attributes` &ndash; The current state of the editor's attributes. It is updated after each editor attributes update. Note that you should not use it if you disabled two-way binding by passing the `disableTwoWayDataBinding` property.
+* `setAttributes` &ndash; The function used for updating the editor's attributes.
+
+## Context feature
+
+The `useMultiRootEditor` hook also supports the {@link features/context-and-collaboration-features context feature}, as described in the main {@link installation/integrations/react#context-feature React integration} guide.
+
+However, as the multi-root editor addresses most use cases of the context feature, consider if you need to employ it.
+
+## Two-way data binding
+
+By default, the two-way data binding is enabled. It means that every change done in the editor is automatically applied in the `data` object returned from `useMultiEditorHook`. Additionally, if you want to change or set data in the editor, you do not need to use the editor API, instead, the hook provides the `setData` method. It works the same way in case of attributes &ndash; the hook provides the `attributes` object and the `setAttributes` method to update them. It ensures that if you want to use or save the state of the editor, these objects are always up-to-date.
```

Comment:
```suggestion
By default, the two-way data binding is enabled. It means that every change done in the editor is automatically applied in the `data` object returned from `useMultiEditorHook`. Additionally, if you want to change or set data in the editor, you can simply use `setData` method provided by the hook. It works the same way in case of attributes &ndash; the hook provides the `attributes` object and the `setAttributes` method to update them. It ensures that if you want to use or save the state of the editor, these objects are always up-to-date.
```